spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/resource/JobInstanceResource.java

```diff
@@ -29,12 +30,16 @@
  */
 public class JobInstanceResource extends ResourceSupport {
 
-	private final String jobName;
+	private String jobName;
 
-	private final long jobInstanceId;
+	private long jobInstanceId;
 
 	private List<JobExecutionResource> jobExecutions;
 
+	public JobInstanceResource(){
```

Comment:
can this be made non-public. Also, document why such a constructor exists (jackson deser)